assets/js/googlesitekit/datastore/site/first-party-mode.js

```diff
@@ -118,7 +118,9 @@ const baseActions = {
 			);
 
 		if ( results?.response?.isEnabled ) {
-			dispatch( CORE_USER ).triggerSurvey( 'fpm_setup_completed' );
+			yield commonActions.await(
+				dispatch( CORE_USER ).triggerSurvey( 'fpm_setup_completed' )
+			);
```

Comment:
While this seems unrelated, it fixes stability issues in other tests where `submitChanges` is awaited where additional arbitrary waits were necessary.